packages/cfd/src/features/Containers/cfd-password-modal/cfd-password-modal.tsx

```diff
@@ -0,0 +1,568 @@
+import React from 'react';
+import { FormikErrors } from 'formik';
+import { useHistory } from 'react-router';
+import { SentEmailModal } from '@deriv/account';
+import {
+    getDxCompanies,
+    getMtCompanies,
+    getDerivezCompanies,
+    TMtCompanies,
+    TDxCompanies,
+    TDerivezCompanies,
+} from '../../../Stores/Modules/CFD/Helpers/cfd-config';
+import { MobileDialog, Modal } from '@deriv/components';
+import {
+    getAuthenticationStatusInfo,
+    getCFDPlatformLabel,
+    getErrorMessages,
+    getFormattedJurisdictionCode,
+    isDesktop,
+    isMobile,
+    routes,
+    validLength,
+    validPassword,
+} from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+import {
+    useAccountStatus,
+    useAvailableMT5Accounts,
+    useCreateMT5Account,
+    useCreateOtherCFDAccount,
+    useSettings,
+    useTradingPlatformPasswordChange,
+    useVerifyEmail,
+} from '@deriv/api';
+import SuccessDialog from '../../../Components/success-dialog.jsx';
+import '../../../sass/cfd.scss';
+import './cfd-password-modal.scss';
+import { observer, useStore } from '@deriv/stores';
+import { useCfdStore } from '../../../Stores/Modules/CFD/Helpers/useCfdStores';
+import { PasswordModalHeader } from './password-modal-header';
+import { CFDPasswordForm } from './cfd-password-form';
+import { IconType } from './icon-type';
+import { TCFDPasswordFormValues, TOnSubmitPassword } from './types';
+import { CFD_PLATFORMS, CATEGORY, JURISDICTION, MARKET_TYPE } from '../../../Helpers/cfd-config';
+
+type TReviewMsgForMT5 = {
+    is_selected_mt5_verified: boolean;
+    jurisdiction_selected_shortcode: string;
```

Comment:
Proper typing from api here !

Reply:
For now we are using the store and unless we get that from api the `seychelles` is still there so its either omit the type which again is the same thing as typing everything.